src/Propel/Runtime/Connection/ConnectionFactory.php

```diff
@@ -55,6 +55,11 @@ public static function create(array $configuration = [], AdapterInterface $adapt
                     }
                     $value = constant($value);
                 }
+                
+                if ($option == "ATTR_EMULATE_PREPARES" || $option == "ATTR_TIMEOUT") {
```

Comment:
They was added automatically in propel https://github.com/propelorm/Propel2/blob/97dcdae4aff3ae6ae5f5bcf356a72babaa9f293d/src/Propel/Common/Config/PropelConfiguration.php#L132